libnavigation-core/src/main/java/com/mapbox/navigation/core/preview/RoutesPreviewUpdateReasons.kt

```diff
@@ -0,0 +1,34 @@
+package com.mapbox.navigation.core.preview
+
+import androidx.annotation.StringDef
+
+/**
+ * All reasons which can cause routes preview update.
+ * @see [RoutesPreviewObserver]
+ */
+object RoutesPreviewUpdateReasons {
+    /**
+     * Routes for preview were set by user.
+     * [RoutesPreviewUpdate.routesPreview] always has value for this reason.
+     * @see [RoutesPreviewObserver]
+     */
+    const val PREVIEW_NEW = "PREVIEW_SET"
```

Comment:
it can be reason, I just wanted to keep things simple. Do you know any scenarios when it's important to distinguish new routes from changing primary route?